main.py

```diff
@@ -13,7 +13,7 @@
 
 users = {}
 notes = {}
-secret_key = os.environ["PASTEBIN_JWT_SECRET"]
+secret_key = 'secret'#os.environ["PASTEBIN_JWT_SECRET"]
```

Comment:
In order for Trufflehog to detect this as a Github secret, change the variable name to something like `github_secret` and make the value a SHA256 hash like `123963c1434c377428ca8531fbc46c0c43d037a1`

Reply:
This is taken care. Secret is coming from CodeSpaces secret